src/lib/swapper/swappers/LifiSwapper/utils/processGasCosts/processGasCosts.ts

```diff
@@ -20,22 +19,20 @@ const processNetworkFee = ({
   return networkFeeCryptoLifiPrecision
 }
 
-const processTradeFee = ({
+const getOtherGasCosts = ({
   allRouteGasCosts,
-  initialSellAssetTradeFeeUsd,
   feeAsset,
 }: {
   allRouteGasCosts: GasCost[]
-  initialSellAssetTradeFeeUsd: BigNumber
   feeAsset: Asset
 }) => {
   const feeAssetAddress = getEvmAssetAddress(feeAsset)
 
-  const nonFeeAssetGasCosts = allRouteGasCosts
-    .filter(gasCost => gasCost.token.address !== feeAssetAddress)
-    .reduce((acc, gasCost) => acc.plus(bnOrZero(gasCost.amountUSD)), bn(0))
+  const nonFeeAssetGasCosts = allRouteGasCosts.filter(
+    gasCost => gasCost.token.address !== feeAssetAddress,
```

Comment:
good idea, i've applied the `Address` type suggestion and renamed the funtion to `getLifiEvmAssetAddress` since it's really returning an EVM address in the world of lifi SDK.